src/client/Microsoft.Identity.Client/Internal/Broker/BrokerInteractiveRequest.cs

```diff
@@ -10,47 +10,85 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace Microsoft.Identity.Client.Internal.Broker
 {
-    internal class BrokerInteractiveRequest
+    internal class BrokerInteractiveRequest : InteractiveRequest
```

Comment:
When you derive from RequestBase, check what methods you should be overrideing and what methods you just need to call.